src/components/Events/newEventMain.js

```diff
@@ -35,7 +36,13 @@ export class NewEventMain extends React.Component {
     this.setState({pageCount: this.state.pageCount - 1})
   }
 
+  goHome = () => {
+    this.setState({redirect: true})
```

Comment:
can just set {pageCount: 0} in this function, and it will redirect to dashboard.  This would clean it up a bit since it already has a redirect case in line 50